code/__DEFINES/dcs/signals.dm

```diff
@@ -219,6 +219,8 @@
 	#define COMPONENT_NO_MOUSEDROP (1<<0)
 ///from base of atom/MouseDrop_T: (/atom/from, /mob/user)
 #define COMSIG_MOUSEDROPPED_ONTO "mousedropped_onto"
+///from base of mob/MouseWheelOn(): (delta_x, delta_y, params)
+#define COMSIG_MOUSE_SCROLL "mousescroll"
```

Comment:
```suggestion
///from base of mob/MouseWheelOn(): (/atom, delta_x, delta_y, params)
#define COMSIG_MOUSE_SCROLL_ONB "mousescroll_on"
```
This signal has an atom component to it. Even if it is not useful for your current purposes it must be added. This is a generic event and must be treated like that, other things will have different requirements than you currently.